packages/rich-list-core/README.md

```diff
@@ -0,0 +1,9 @@
+# @defichain/rich-list-core
+Complementary library module for rich-list-api to compute and keep a rich list state in injected storage service.
+
+## Prerequisite
+Three dependencies to be injected
```

Comment:
I think you meant two?

Reply:
Yeah, didn't change it after change the lib components 😅